worlds/soccer/Robot Shape.obj

```diff
@@ -0,0 +1,193 @@
+# WaveFront *.obj file (generated by Autodesk ATF)
+
+mtllib Robot Shape.mtl
```

Comment:
@resetteam2023 My first question regarding the shape of the robot is: Would it be possible to change the cube shape into cylindrical one?